service/middleware.go

```diff
@@ -30,9 +30,7 @@ func requestIDMiddleware() mux.MiddlewareFunc {
 	}
 }
 
-var allowedOrigins = []string{"*.observableusercontent.com", "localhost"}
```

Comment:
Note that I've removed these baked-in values!

Reply:
Cool, we need to make sure brim uses this new command argument to add these origins.

Reply:
I think it'll work fine without them.

We will however need to add https://*.observableusercontent.com on cloud deployments to keep the Observable integration working.

Reply:
Oh ok, good point.